charts/kyuubi/values.yaml

```diff
@@ -115,6 +133,12 @@ server:
       port: "{{ .Values.server.mysql.port }}"
       nodePort: ~
       annotations: {}
+      #candidates are ClientIP or None, ClientIP is required to set if session affinity is enabled
+      sessionAffinity: ~
+      #default is 10800s
+      sessionAffinityConfig:
+        clientIP:
+          timeoutSeconds: ~
```

Comment:
this could be flexible once K8s supports other session affinity strategies. let's use toYaml approach

Reply:
sounds good. Thanks

Reply:
> this could be flexible once K8s supports other session affinity strategies. let's use toYaml approach

Refactored. Thanks